src/components/app/components/banner.js

```diff
@@ -1,58 +1,29 @@
-                                
-                                </p></h5>
-                            </div>
+const { BANNER, FLEXBOX, BANNER_CARD, BANNER_ICON, BANNER_COLOR } = BANNER_CLASSNAMES;
 
-                            <div className="box" style={{width: '200px', display: 'block', textAlign: 'center',  padding: '15px',}}>
-                                <div style={{borderRadius:'15px', width: '50px', height: '50px', backgroundColor: 'white', margin: 'auto',}}>
-                                    <i style={{ color: '#106021', fontSize: '20px', paddingTop: '15px', }} className="fa fa-trash"></i>
-                                </div>
-                                <h3 style={{color: 'white',}}>Reduce Food Wastage</h3>
-                                <h5 style={{ color: 'white',}}><p> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
-                                tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
-                                
-                                </p></h5>
-                            </div>
+const banners = BANNER_DATA.map(({ key, icon, header, body }) => (
+    <div className={BANNER_CARD} key={key}>
+        <div className={BANNER_ICON}>
+            <i className={icon} ></i>
+        </div>
+        <h3 className={BANNER_COLOR}>{header}</h3>
+        <h5 className={BANNER_COLOR}>{body}</h5> 
+    </div> 
+)); 
 
-                            <div className="box" style={{width: '200px', display: 'block', textAlign: 'center',  padding: '15px',}}>
-                                <div style={{borderRadius:'15px', width: '50px', height: '50px', backgroundColor: 'white', margin: 'auto',}}>
-                                     <i style={{ color: '#106021', fontSize: '20px', paddingTop: '15px', }} className="fa fa-users"></i>
-                                </div>
-                                <h3 style={{color: 'white',}}>Social Economic Driven</h3>
-                                <h5 style={{ color: 'white',}}><p> Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
-                                tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
-                                
-                                </p></h5>
-                            </div>
-                        </div>
-                        <Button>Learn More</Button>
-                    </div>
+class Banner extends Component { 
+    render() {
+        return (
+            <div className={BANNER}>
+                <h2>{BANNER_HEADING}</h2>
+                <div className={FLEXBOX}>{banners}</div>
```

Comment:
Do not name this className flexbox. Name is `banners` or `banner-container` given that it contains the banners`